engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/sql/HsqlSqlQueryableDataBase.scala

```diff
@@ -20,8 +22,14 @@ class HsqlSqlQueryableDataBase(query: String, tables: Map[String, ColumnModel])
 
   import HsqlSqlQueryableDataBase._
 
-  private val connection: Connection =
-    DriverManager.getConnection(s"jdbc:hsqldb:mem:${UUID.randomUUID()};shutdown=true;allow_empty_batch=true", "SA", "")
+  private val connection: Connection = {
+    val url = s"jdbc:hsqldb:mem:${UUID.randomUUID()};shutdown=true;allow_empty_batch=true"
+    //TODO: should not really happen, but sometimes happens in tests..
```

Comment:
Why it happens? Is there some kind of race? Can u be more specific here? Also can you catch more concrete exceptions e.g. using `catching[...] opt` syntax?

Reply:
I'll extract it to another PR as it's not scala 2.12 related...

Reply:
Ok, so please only add notice that it is because some race condition or what u find out during work on this PR...

Reply:
I would like to leave it with `Try` and description as it was instead of just removing it :-D But your choice